packages/xo-web/src/xo-app/backup-ng/new/_reportWhen.js

```diff
@@ -0,0 +1,72 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import PropTypes from 'prop-types'
+import React from 'react'
+import Icon from 'icon'
+import Select from 'form/select'
+import Link from 'link'
+import Tooltip from 'tooltip'
+import { generateId } from 'reaclette-utils'
+import { injectState, provideState } from 'reaclette'
+
+import { FormGroup } from './../utils'
+
+const REPORT_WHEN_FILTER_OPTIONS = [
+  {
+    label: 'reportWhenAlways',
+    value: 'always',
+  },
+  {
+    label: 'reportWhenFailure',
+    value: 'failure',
+  },
+  {
+    label: 'reportWhenNever',
+    value: 'Never',
+  },
+]
+
+const getOptionRenderer = ({ label }) => <span>{_(label)}</span>
+
+const ReportWhen = decorate([
+  provideState({
+    computed: {
+      idInput: generateId,
+    },
+  }),
+  injectState,
+  ({ state, onChange, value }) => (
+    <FormGroup>
+      <label htmlFor={state.idInput}>
+        <strong>{_('reportWhen')}</strong>
+      </label>{' '}
+      <Tooltip content={_('pluginsWarning')}>
```

Comment:
We can disable `Select` when these plugins aren't loaded?
What if they aren't loaded and we create a backup with `Report when`   
it'll be taking into count?

Reply:
The report when property is used on the backup job execution. It will not have any side effect when creating/editing a backup job.